src/components/RoomDropdown/index.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useEffect, useRef } from 'react'
+import { useState } from 'react'
+import styled from 'styled-components'
+import { KARELIA } from '../../utils/css'
+
+type Props = {
+    button?: any
+}
+
+const DropDownContainer = styled.div`
+    margin-left: 10px;
+`
+const DropDownHeader = styled.div`
+    margin-bottom: 0.8em;
+    padding: 0 2em 0 1em;
+    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.15);
+    font-weight: 500;
+    font-size: 1.3rem;
+    color: #3faffa;
+    background: #ffffff;
+`
+const DropDownListContainer = styled.div`
+    position: absolute;
+    top: 75px;
+    right: 35px;
+    z-index: 100;
+`
+const DropDownList = styled.table`
```

Comment:
The list is a `table`. I don't feel very good about using `tr` element as a button or a link. Doesn't feel semantically right.

Usually a structure like `ul > li > Clickable` is prefered. The `Clickable` is a link or a button, includes a designated place for the icon and the text.

Reply:
The reasoning behind this was that the text didn't line up together. By using a table, the icons and text where on separate columns so they could both be aligned properly. I'm not sure how to use a list and also have the text line up. Do you have a suggestion for this?

Reply:
Naturally, at least 2 ideas:
- flex box (used in `SidebarItem`)
- `padding-left` for text and the icon positioned absolutely (I used it for special cases)

The trick is to implement the item component with a fixed layout. Have a look at my [`SidebarItem`](https://github.com/streamr-dev/streamr-chat/blob/development/src/components/pages/Chat/SidebarItem.tsx). It's a very similar structure.